dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/internal/JDBCDriverService.java

```diff
@@ -377,6 +377,11 @@ public Object createAnyDataSourceOrDriver(Properties props) throws SQLException
             if (nonshipFunction) {
                 String url = props.getProperty("URL", props.getProperty("url"));
                 if (url != null) {
+                    if(url.toLowerCase().contains("logintimeout") || props.containsKey("loginTimeout")) {
+                        SQLNonTransientException failure = connectorSvc.ignoreWarnOrFail(tc, null, SQLNonTransientException.class, "INVALID_LOGINTIMEOUT", dataSourceID);
```

Comment:
ignoreWarnOrFail is not appropriate here unless you are able to parse the loginTimeout out of the URL